template/cba_settings.sqf

```diff
@@ -1,8 +1,18 @@
 // TAC Settings
-force tac_apollo_enabled = true;               // Set to 'false' to disable persistence for missions where we aren't using our own gear.
+force tac_apollo_enabled = false;               // Set to 'false' to disable persistence for missions where we aren't using our own gear.
 
 // ACE Settings
 force ace_respawn_savePreDeathGear = false;   // Save and respawn with pre-set gear upon death. (NCO ONLY)
```

Comment:
```suggestion
force ace_respawn_savePreDeathGear = true;   // Save and respawn with pre-set gear upon death. (NCO ONLY)
```

If by default with the template tac_apollo is off, this should be on.

Reply:
JJ wanted to see the other settings like this that have been changed

Reply:
Yeah, I would enable this for SOG template.